swift/src/IceImpl/Communicator.mm

```diff
@@ -379,6 +379,8 @@ - (nullable dispatch_queue_t)getClientDispatchQueue:(NSError* _Nullable* _Nullab
 {
     try
     {
+        // Swift always sets InitializationData.useDispatchQueueExecutor to true
+        assert(self.communicator->getClientDispatchQueue());
```

Comment:
Since it's always non-null, why do we return a nullable? For the error?

Reply:
Yes, it's required for the error.